stable/postgresql/values.yaml

```diff
@@ -154,3 +154,6 @@ podAnnotations: {}
 ## Deployment pods replace strategy
 ## ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
 # strategy: {}
+
+# Define custom environment variables to pass to the image here
+env: {}
```

Comment:
`extraEnv ` should be used and also please add this new variable in the readme.md

Reply:
> extraEnv should be used and also please add this new variable in the readme.md

Sure, we could do that, but seems non-consistent. Other charts in this same repository use the `env` syntax, e.g.
https://github.com/helm/charts/blob/4dd8cdc6bde7701c96ceb171f3ac3e2c9f521efd/stable/sentry/templates/web-deployment.yaml#L99

Is there an official policy on this?

Reply:
can use env or extraEnv, but since this is extra variables I would prefer to use that. 
I was searching the docs and did not find, but one of the maintainers of the charts reviewed my PR some days ago and asked to use extraEnv as well: https://github.com/helm/charts/pull/7126#discussion_r210263817

Reply:
Sure, let's go with `extraEnv`then. I'd suggest reviewing other charts and setting a policy then to have everything in official repo as consistent as possible.

Reply:
yes agree. cc @davidkarlsen @mattfarina @unguiculus